File: Python3_CPython/runner.py
from typing import List
import config
import rl


def run(argv: List[str]):
    cfg = config.Config(argv)
    rl_ = rl.RL(cfg)
    rl_.run()
    rl_.save_returns()
    rl_.run_test()
    rl_.save_test_history()

File: Ruby_MRI_JIT/runner.rb
require_relative "config"
require_relative "rl"

def run(argv)
  if argv.length != 1
    raise "invalid argv length: #{argv}"
  end

  config = Config.new(argv[0])
  rl = RL.new(config)
  rl.run
  rl.save_returns
  rl.run_test
  rl.save_test_history
end
